src/App.js

```diff
@@ -94,22 +95,20 @@ class App extends Component {
   render() {
     return (
       <div className="App">
-        <header className="App-header">
-          {this.state.user ? <HeaderNav user={this.state.user} /> : null}
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            <FormattedHTMLMessage id="app.title" />
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            <FormattedHTMLMessage id="app.link" />
-          </a>
-          <p>{this.state.user ? this.state.user.Email : ''}</p>
-        </header>
+        {this.state.user ? <Header className="App-header" user={this.state.user} /> : null}
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          <FormattedHTMLMessage id="app.title" />
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <FormattedHTMLMessage id="app.link" />
```

Comment:
Is it good practice? I mean, as, instinct I try to avoid injecting HTML always I can.

Reply:
`FormattedHTMLMessage` is part of how it works react intl library, not a html text but a resourced text.

Reply:
mmm, and when we are sure that it is plain text we should use `FormattedHTMLMessage` anyway?

Reply:
Oh, I understand, it is only for _react-intl_. There is a nice description about the posibilities in https://github.com/yahoo/react-intl/wiki/Components

Based on what that page says, what do you think about using `FormattedMessage` in place of `FormattedHTMLMessage`?

Reply:
Maybe it should be used `FormattedMessage` instead. Html is for html tags as you said, the resource here `app.link` has a text.

Reply:
@fcoronelmakingsense please check non html resources should use `FormattedMessage` instead `FormattedHTMLMessage`

Reply:
> Html is for html tags as you said

I think that `FormattedMessage` also support simple HTML formatting, I am not sure, but I understood that.

Reply:
Hi, i didn't know about `FormattedMessage` i can research about this